packages/venia-ui/lib/components/CartPage/GiftCards/giftCards.js

```diff
@@ -132,6 +123,17 @@ const GiftCards = props => {
                         validate={isRequired}
                     />
                 </div>
+                {shouldDisplayCardBalance && (
```

Comment:
Consider extracting to constant to keep JSX depth accurate.

Reply:
Done in b87b2e757